docs/reference/TYPESCRIPT_ANY_FIXES.md

```
@@ -0,0 +1,403 @@
+**Current Issues (Lines 248-250, 352, etc.):**
+```typescript
+// ❌ Current
+const transformResident = (resident: any): any => {
+  return {
+    // transformation logic
+  };
+};
+```
+
+**Fix:**
+```typescript
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)